rollup.config.js

```diff
@@ -94,10 +93,6 @@ export default (cliArgs) => [{
     // Watch the `public` directory and refresh the
     // browser on changes when not in production
     !production && livereload("public"),
-
-    // If we're building for production (npm run build
-    // instead of npm run dev), minify
-    production && terser(),
```

Comment:
What if, instead of removing the plugin, we only disabled mangling?

```js
    // If we're building for production (npm run build
    // instead of npm run dev), minify
    production && terser({
      mangle: false,
    }),
```

Reply:
See my comment [here](https://github.com/mozilla-rally/rally-core-addon/issues/330#issuecomment-781966238)

Reply:
Yeah, I didn't perform any due dilligence here :) much better strategy. I'll add that option.